src/ai/backend/client/output/json.py

```diff
@@ -122,6 +123,52 @@ def print_paginated_list(
             **_json_opts,
         ))
 
+    def print_mutation_result(
+        self,
+        item: Mapping[str, Any],
+        item_name: Optional[str] = None,
+        action_name: Optional[str] = None,
+        extra_info: Mapping = {},
+    ) -> None:
+        t = {
```

Comment:
Let's use a more sensible variable name like `data` in both `print_mutation_result()` and `print_mutation_error()`.